twopow.c

```diff
@@ -0,0 +1,17 @@
+//Anya Lauria
```

Comment:
Nice use of the do while loop, but your program didn't include 2^0 = 1. Be careful with these cases.